app/views/idv/in_person/verify_info/show.html.erb

```diff
@@ -160,7 +160,7 @@ locals:
                 poll_interval_ms: IdentityConfig.store.poll_rate_for_verify_in_seconds * 1000,
               },
             },
-          ).with_content(t('forms.buttons.continue')) %>
+          ).with_content(t('forms.buttons.submit.default')) %>
```

Comment:
The ticket doesn't say to change the In person verify info submit button as far as I saw. Did we decide to keep just the buttons in sync?